source/mongodump.txt

```diff
@@ -60,6 +60,66 @@ Run ``mongodump`` from the system command line, not the
 
 .. include:: /includes/fact-online-archive-callout.rst
 
+``mongodump`` exports:
```

Comment:
nit: can we choose a different word than export, e.g., "capture" or "dump"? I think the export language may make this more easily confused with mongoexport.

Reply:
Will do.